sopel/irc/backends.py

```diff
@@ -80,6 +84,21 @@ def __init__(
         self._keyfile: Optional[str] = keyfile
         self._verify_ssl: bool = verify_ssl
         self._ca_certs: Optional[str] = ca_certs
+        self._ssl_ciphers = ":".join(ssl_ciphers)
+
+        self._ssl_minimum_version = ssl_minimum_version
+        if isinstance(ssl_minimum_version, str):
```

Comment:
I'd rather invert the if here:

```suggestion
        if not isinstance(ssl_minimum_version, ssl.TLSVersion):
```